src/Sequence.ts

```diff
@@ -526,6 +528,35 @@ export class Sequence<TItem> implements Iterable<TItem> {
     return new Sequence(createTakeWhileIterable(this._iterable, predicate));
   }
 
+  /**
+   * Returns elements from a sequence as long as they don't exist in the specified iterable items.
+   *
+   * @param items     The provided set of items that should not be in the returned Sequence.
+   * @param predicate The optional predicate that determines if two TItem items are equal.
+   * @returns Sequence<TItem>
```

Comment:
This could be omitted as it only contains the type, which could be set using TypeScript

```suggestion
```

Reply:
This is an old habit of mine, sadly! In any case, I can remove it.